src/ds/d400/d400-options.h

```diff
@@ -11,7 +11,7 @@
 #include "../../hdr-config.h"
 
 #include <rsutils/lazy.h>
-
+#include <src/hid-sensor.h>
```

Comment:
Can be removed from the header as we use only pointers here

Reply:
I get errors without it that I use undefined type

Reply:
Fixed